Gordon360/Services/RecIMService.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Gordon360.Models.CCT;
+using Gordon360.Models.CCT.Context;
+using Gordon360.Models.ViewModels;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Gordon360.Services
+{
+    public class RecIMService : IRecIMService
+    {
+        private readonly CCTContext _context;
+
+        public RecIMService(CCTContext context)
+        {
+            _context = context;
+        }
+
+     
+        public IEnumerable<League> GetLeagues()
+        {
+            var leagues = _context.League.AsEnumerable();
+            System.Diagnostics.Debug.WriteLine(leagues);
+            return leagues;
+        }
+        public async Task PostLeague(League newLeague)
```

Comment:
You can remove async here

Reply:
I'll do that, I still have a bad habit of pre-emptive coding where I imagined that I would need an await call inside.